admin/scripts/zvlastni/reporty/stravenky.xtpl

```diff
@@ -1,20 +1,27 @@
 <!-- begin:stravenky -->
 <style>
 
-.radek { border: solid 0.3mm; page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
-.radek > div { float: left; width: 16mm; height: 16mm; padding: 2mm; border-left: solid 0.3mm; overflow: hidden; position: relative; }
+.radek { page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
+.radek > div { float: left; width: 22mm; height: 22mm; padding: 2mm; box-shadow:
+    1px 0 0 0,
+    0 1px 0 0,
+    1px 1px 0 0,   /* Just to fix the corner */
+    1px 0 0 0 inset,
+    0 1px 0 0 inset; overflow: hidden; position: relative; }
```

Comment:
Řešení přes box-shadow není ok, mělo by se to předělat na border. Imho to jde udělat i aby to vypadalo hezky (různým laděním na jakých stranách mají bordery být), ale i kdyby to nešlo, tak pořád lepší aspoň ať se to dá v tom stylu pochopit.